torch_xla/csrc/helpers.h

```diff
@@ -140,6 +140,14 @@ class XlaHelpers {
 
   static xla::XlaComputation CreateMaxComputation(xla::PrimitiveType type);
 
+  // Returns an XLA operation which is a reshape to the expected rank, by
+  // appending 1s to the major dimension. If offset is greater than zero, 1s
+  // will be prepened to the minor dimension as well.
+  // Expected condition: rank(input) + offset <= expected_rank
+  static xla::XlaOp NormalizeShape(const xla::XlaOp& input,
```

Comment:
I think `ReshapeToRank` would be a better name.

Reply:
SG